akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/PostgresOffsetStoreDao.scala

```diff
@@ -42,13 +53,16 @@ private[projection] class PostgresOffsetStoreDao(
     projectionId: ProjectionId)
     extends OffsetStoreDao {
 
-  private val logger = LoggerFactory.getLogger(getClass)
+  protected def logger: Logger = LoggerFactory.getLogger(getClass)
```

Comment:
could you do the same as in a-p-r2dbc to avoid accessing LoggerFactory each time:
```
protected def log: Logger = PostgresOffsetStoreDao.log
```

Reply:
should `PostgresOffsetStoreDao.log` then be a `val`, or what is the purpose?

Reply:
Yes, so that it's created once instead of for each `log.x` call

Reply:
Ok, makes sense. Then I think I made it wrong there as well: https://github.com/akka/akka-persistence-r2dbc/blob/main/core/src/main/scala/akka/persistence/r2dbc/internal/sqlserver/SqlServerDurableStateDao.scala#L33. Will fix (if possible).